sdk/python/feast/job_service.py

```diff
@@ -117,6 +126,22 @@ def StartStreamToOnlineIngestionJob(
         feature_table = self.client.get_feature_table(
             request.table_name, request.project
         )
+
+        if self.client._config.getboolean(CONFIG_JOB_SERVICE_ENABLE_CONTROL_LOOP):
```

Comment:
I'm looking at the code now and just wondering if it shouldn't be restructured in some ways. We have two approaches.
1. Keep as much logic and decision-making within the client. Make the Job Service a thin wrapper on the client. Don't break abstraction by accessing internal logic within the client.
2. Make the service/client modes operate at the same abstraction, but then don't use the client at all (create our own config objects, access methods directly).

@tsotnet it seems like this implementation mixes these two approaches. We create a `client` and use it for configuration, but then import internal methods (PySpark launcher and job methods) directly.